workflow/executor.py

```diff
@@ -185,6 +186,8 @@ def _inflate_sources(self, logger: logging.Logger, rai_config: RaiConfig, env_co
             inflated_paths = self.paths_builders[src.container.name].build(logger, days, src.relative_path,
                                                                            src.extensions,
                                                                            src.is_date_partitioned)
+            if src.is_support_data_size_print():
```

Comment:
```suggestion
            if src.has_size_attribute():
```
maybe this way better?

Reply:
size is not a source attribute

Reply:
@stanislav-bedunkevich maybe `is_size_supported()`